AppCenter/AppCenterTests/MSACAbstractLogTests.m

```diff
@@ -159,6 +163,14 @@ - (void)testIsEqual {
 
   // Then
   XCTAssertFalse([self.sut isEqual:log]);
+
+  // When 
+  self.sut.userId = @"FAKE-USER-ID";
```

Comment:
remove userid

Reply:
Leave as is, because this value reverts previous test value.